test/commands/runtime/action/list.test.js

```diff
@@ -175,5 +175,32 @@ describe('instance methods', () => {
           expect(stdout.output).toMatchFixture('action/list-name-sort-output.txt')
         })
     })
+
+    test('return action count', () => {
```

Comment:
maybe one more test case for count = 0.
same for all other tests for package, activations etc.

Reply:
I pushed a comment to add this test. Thanks @sandeep-paliwal.